### ISSUE TITLE
Plugins can no longer be `npm link`'d

### ISSUE THREAD

**Issue by sindresorhus** [wulkano/kap#702]
In Kap v2, you could `npm link` a plugin into the plugins directory during development, which is very handy, and also [documented](https://github.com/wulkano/kap/blob/master/docs/plugins.md#development). In v3, there's just an ENOENT error that it can't find the plugin in `node_modules`. Somewhere we don't seem to be resolving symlinks properly.

Also, I saw errors about the plugin not being published, since it's in package.json, we need to improve that somehow as it's not a good experience when developing a plugin before actually publishing it.

**Comment by karaggeorge** [comment]
I think I fixed part of this in #700. One of the issues I saw was that after we do `npm link {plugin name}` and then start Kap, when the app starts it does a `npm prune` and `npm install`, which causes that error about the plugin not being published. I moved those two calls to not happen in development (`npm install` still happens when you install a new plugin)

I haven't encountered the first error. I was able to link a plugin and test the `kapVersion` thing after removing those two calls
